### ISSUE TITLE
Accordion on mobile only?

### ISSUE THREAD

**Issue by TrilipuT** [uikit/uikit#3550]
Hey !
I need to add accordion only on mobile (let's say <768px) and show fully expanded content on larger screens. I can't find such option in the accordion component.
I see only JS way to achieve this - initialize and destroy accordion based on screen size.
Is there any way to do this out of the box?

But maybe you can add an option like `media` in toggle component? 
Something like `<ul uk-accordion="media: @m">`
Would be great to indicate max or min. For ex 
`media-max: @m` - initialize on sizes less than **m**
`media-min: @m` - initialize on sizes more that **m**

**Comment by psuplat** [comment]
By "fully expanded" I assume you mean that all accordion elements are opened, while on mobile they are all closed by default.

The quickest way to achieve what you want is to have two accordions, one visible on mobiles only the other on everything else. The general template would look like this:

```
<!-- visible on anything bigger then mobile -->
<ul uk-accordion="multiple: true" class="uk-text-primary uk-visible@s">
    <li class="uk-open"></li>
    <li class="uk-open"></li>
    <li class="uk-open"></li>
</ul>

<!-- visible only on mobile -->
<ul uk-accordion="multiple: true" class="uk-hidden@s">
    <li></li>
    <li></li>
    <li></li>
</ul>
```

See this: http://jsfiddle.net/pk0zdcn4/8/

**Comment by TrilipuT** [comment]
Yep, that will work. Thanks for idea @psuplat . Just  in my case i don't need accordion on large screens at all.

But still would be nice to have ability to set on what screen size accordion should be initialized.

**Comment by PixelT** [comment]
+1 for this
Food example for usage of this is visible content on desktop, where on mobile / tablet the content by default is collapse
@saschadube do you plan to add this option?

**Comment by TrilipuT** [comment]
I'm trying to `display: block !important;` on desktop but I still have a clickable accordion title.
Also, I'm thinking about initializing accordion in JS at certain media queue.

But having 2 accordions is the only option now... @saschadube can you suggest here?

**Comment by psuplat** [comment]
I think you are completely missing the purpose of the accordion element. The header is a link element `<a>` so it will always be clickable, unless you add some custom JS to control this, but then what's the point in doing this.
Like I mentioned before, use "Description List" instead - that's pretty much always-open, un-collapsible version of accordion.

There is no need to add `display: block !important;` to accordion content. Just add class `uk-open` to the `<li>` element, and set the accordion to multiple: true in `<ul>`, like this:
```
<ul uk-accordion="multiple: true">
    <li class="uk-open">
        <a class="uk-accordion-title" href="#">Item 1</a>
        <div class="uk-accordion-content">
            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
        </div>
    </li>
    <li class="uk-open">
        <a class="uk-accordion-title" href="#">Item 2</a>
        <div class="uk-accordion-content">
            <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor reprehenderit.</p>
        </div>
    </li>
    <li class="uk-open">
        <a class="uk-accordion-title" href="#">Item 3</a>
        <div class="uk-accordion-content">
            <p>Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat proident.</p>
        </div>
    </li>
</ul>
```

but, again you would really think your design through - do you really need accordion that is always open and cannot be closed?

**Comment by TrilipuT** [comment]
@psuplat I think you understood me wrong :)
I just need to have accordion only on mobile ( or screen size < 600px or whatever ). The idea was to initialize it only on small screen sizes. Maybe do it in JS.
Right now I'm using your first advice to have separate HTML for mobile and desktop. So i'm showing block with accordion only on mobile. It looks like
`
<ul class="uk-visible@s">
    <li></li>
    <li></li>
    <li></li>
</ul>


<ul uk-accordion class="uk-hidden@s">
    <li></li>
    <li></li>
    <li></li>
</ul>`

But i don't like the idea of duplicating content...

We can think about this problem more widely. For ex add for all features parameter when to initialize. I understand that this is a big piece of work.

**Comment by jeannnemelanie** [comment]
+1 for @TrilipuT idea to handle it « more widely and add for all features parameter ». In my case, I need to show my content in a Slider in mobile view, but nothing except content in desktop view.
